specificatie/genereervariant/openapi.json

```diff
@@ -2,15 +2,15 @@
   "openapi" : "3.0.0",
   "info" : {
     "title" : "Bevragen Ingeschreven Personen",
-    "description" : "API voor het bevragen van ingeschreven personen uit de basisregistratie personen (BRP), inclusief de registratie niet-ingezeten (RNI). Met deze API kun je personen zoeken en actuele gegevens over personen, kinderen, partners en ouders raadplegen.\n\nGegevens die er niet zijn of niet actueel zijn krijg je niet terug. Heeft een persoon bijvoorbeeld geen geldige nationaliteit, en alleen een beëindigd partnerschap, dan krijg je geen gegevens over nationaliteit en partner.\n\nZie de [Functionele documentatie](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v1.0.0/features) voor nadere toelichting.\n",
+    "description" : "API voor het bevragen van ingeschreven personen uit de basisregistratie personen (BRP), inclusief de registratie niet-ingezeten (RNI). Met deze API kun je personen zoeken en actuele gegevens over personen, kinderen, partners en ouders raadplegen.\n\nGegevens die er niet zijn of niet actueel zijn krijg je niet terug. Heeft een persoon bijvoorbeeld geen geldige nationaliteit, en alleen een beëindigd partnerschap, dan krijg je geen gegevens over nationaliteit en partner.\n\nZie de [Functionele documentatie](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v1.1.0/features) voor nadere toelichting.\n",
```

Comment:
hier zit weer de \n aan het eind. Heb je de genereervariant lokaal gegenereerd?

Reply:
Volgens mij horen deze \n er in de json variant gewoon in. Er staan er heel erg veel in en als ik het bestand in "RAW"-formaat bekijk zijn ze weg. Vooralsnog ben ik er van af gebleven.

Reply:
@Melvin, kan jij aangeven op deze \n in de json variant een probleem zijn ? En of ze niet gegenereerd worden dor de resolver op Github ? 
Dit is het laatste puntje voordat ik release v1.1.0 kan maken.

Reply:
De \n komt ook 310 keer voor in de openapi.json van BRK-bevragen. Ik veronderstel dus dat die daarin gewoon thuis horen.

Reply:
de \n in de json variant is geen issue. Ze komen voor in de gegenereerde json variant omdat in json markdown niet wordt ondersteund. Een nieuwe regel in markdown wordt een \n in string